modules/flowable-ui-modeler/flowable-ui-modeler-rest/src/main/java/org/flowable/app/rest/editor/ModelResource.java

```diff
@@ -312,127 +315,23 @@ protected ModelRepresentation createNewModel(String name, String description, In
     return new ModelRepresentation(newModel);
   }
 
-  @RequestMapping(value = "/rest/model/exportForDeploy/{modelId}")
+  @RequestMapping(value = "/rest/models/{modelId}/exportForDeploy")
   @ResponseBody
   public ResponseEntity<Map<String, String>> exportForDeploy(@PathVariable("modelId") String modelId)
-          throws JsonProcessingException, IOException
   {
-
-    Model modelData = modelService.getModel(modelId);
-    JsonNode modelNode = objectMapper.readTree(modelData.getModelEditorJson());
-    byte[] bpmnBytes = null;
+    Model model = modelService.getModel(modelId);
     
-    Map<String, String> formMap = new HashMap<String, String>();
-    Map<String, String> decisionTableMap = new HashMap<String, String>();
-
-    List<Model> referencedModels = modelRepository.findByParentModelId(modelId);
-          for (Model childModel : referencedModels) {
-              if (Model.MODEL_TYPE_FORM_RDS == childModel.getModelType()) {
-        formMap.put(childModel.getId(), childModel.getKey());
-
-              } else if (Model.MODEL_TYPE_DECISION_TABLE == childModel.getModelType()) {
-        decisionTableMap.put(childModel.getId(), childModel.getKey());
-      }
-    }
-          
-    BpmnModel model = new RDSBpmnJsonConverter().convertToBpmnModel(modelNode, formMap, decisionTableMap);
-    formsAsExtension(model);
```

Comment:
nope. I added this code in earlier PR, now it's been refactored into ModelServiceImpl.java